src/Reducer/timeReducer.js

```diff
@@ -1,9 +1,25 @@
+const setBreakTime = (state, action) => {
+	if (action.payload > 0 && action.payload < 61)
+		return { ...state, break: action.payload };
+	else {
+		alert('Time must be between 0 to 60 mins');
```

Comment:
better not to use alert, use can use toast or modal instead , it is considered as bad practice ,
reference : https://stackoverflow.com/questions/8825384/alert-is-bad-really

Reply:
Thanks for the suggestion, I'll replace it with a toast!